### ISSUE TITLE
a11y: add missing alt attribute for decorative images

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#14125]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

`a11y issue 88`
The "alt" attribute is not provided in the source code of the decorative image that resulted in screen reader users accessing the decorative image and communicated as unlabelled.

Account and API Keys
Main Content 
- Decorative image in "Account" section

Create Project
Main content
- Decorative image in "Members" section

Extensions
Main content
- Decorative images associated with "AKS Provisioning", "EKS Provisioning" and so on text.

**To Reproduce**

**Result**
The "alt" attribute is not provided in the source code of the decorative image that resulted in screen reader users accessing the decorative image and communicated as unlabelled.

**Expected Result**

Decorative image should be hidden from screen readers OR provide a meaningful `alt` attribute

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-71a527a9c07db8bcc2ddfe95ae3ed5bd012f4589-head`
UI version: `master 980d4b0`
 Firefox version: `138`
Chrome version: `136`

I wasn't able to navigate with ordinary methods using the keyboard to the decorative images. Using on the other hand dev tools to force their selection I was able to capture the voice over. However with the `alt` text attribute on these images is good enough to meet the acceptance criteria. ✅
